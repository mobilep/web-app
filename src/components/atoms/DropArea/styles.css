.DropArea {
	display: flex;
	height: 260px;
	text-align: center;
	justify-content: center;
	align-items: center;
}

.DropArea-paragraph {
	font-size: 22px;
	line-height: 30px;
	font-weight: var(--semiBold);
	color: var(--black2);
}

.DropArea-paragraph svg {
	position: relative;
	top: 10px;
	width: 42px;
	height: 42px;
	color: var(--tomatoRed);
	margin-right: 20px;
}
