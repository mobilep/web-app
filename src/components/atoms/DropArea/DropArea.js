import React from 'react';
import PropTypes from 'prop-types';
import { FileIcon } from '../../../icons';

import './styles.css';

const DropArea = ({ text }) => (
	<div className='DropArea'>
		<span className='DropArea-paragraph'>
			<FileIcon />
			{text}
		</span>
	</div>
);

DropArea.propTypes = {
	text: PropTypes.string,
};

export default DropArea;
