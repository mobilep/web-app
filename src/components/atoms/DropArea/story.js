import React from 'react';
import { storiesOf } from '@storybook/react';
import DropArea from './DropArea';

storiesOf('1. Atoms / DropArea', module)
	.add('DropArea', () => <DropArea text='Drop files here' />);
