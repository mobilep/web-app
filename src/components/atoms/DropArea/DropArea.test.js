import React from 'react';
import { shallow } from 'enzyme';

import DropArea from './DropArea';

describe('Atom - DropArea', () => {

	it('Should render', () => {
		const wrapper = shallow(<DropArea text='Text' />);
		expect(wrapper).toMatchSnapshot();
	});

});

