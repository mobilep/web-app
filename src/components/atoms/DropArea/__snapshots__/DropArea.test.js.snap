// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - DropArea Should render 1`] = `
<div
  className="DropArea"
>
  <span
    className="DropArea-paragraph"
  >
    <FileIcon />
    Text
  </span>
</div>
`;
